app/src/main/java/io/zenandroid/onlinego/model/ogs/OGSChallengeRequest.kt

```diff
@@ -4,30 +4,27 @@ import io.zenandroid.onlinego.ogs.TimeControl
 
 data class OGSChallengeRequest (
         val initialized: Boolean,
-        val min_ranking: Int = -1000,
-        val max_ranking: Int = 1000,
+        val min_ranking: Int = 0,
+        val max_ranking: Int = 60,
         val challenger_color: String,
-        val aga_ranked: Boolean,
-        val game: Game
-
+        val game: Game,
+        val aga_ranked: Boolean
 ) {
     data class Game(
-            val handicap: String,
-            val time_control: String,
-            val challenger_color: String,
+            val name: String?,
```

Comment:
Please try to keep the order of the fields consistent in future PRs, as it makes the code reviews a lot easier. For example here it's quite hard to see what was added/removed because a lot of the things just changed order...

Reply:
yup, should be a one-time thing - given that I had to edit them, I took the chance to align them to the order used by the OGS FE as it will be easier to compare if needed